src/components/forms/account/AccountForm.tsx

```diff
@@ -285,17 +285,19 @@ async function onSubmit(
 
   if (action === 'add') {
     await account.save();
+    mutate('/api/accounts');
     if (data.balance) {
       await createBalance(data, account);
     }
   } else if (action === 'update') {
     await account.save();
+    mutate('/api/accounts');
   } else if (action === 'delete') {
     await Account.remove(account);
+    mutate('/api/accounts');
     router.replace('/dashboard/accounts');
   }
 
-  mutate('/api/accounts');
```

Comment:
Refreshing accounts this late causes issues with `monthly-totals` taking very long to refresh. I'm... actually not sure why though